src/uct/ib/rdmacm/rdmacm_cm_ep.h

```diff
@@ -0,0 +1,36 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include "rdmacm_listener.h"
+
+typedef struct uct_rdmacm_cep {
+    uct_base_ep_t          super;
+    uct_rdmacm_cm_t        *cm;
+    struct rdma_cm_id      *id;
+    struct ibv_cq          *cq; /* dummy cp used for creating a dummy qp */
```

Comment:
cp -> cq

maybe create a cq on cm and use it for all eps created on that cm to minimize resource utilization?

Reply:
@yosefe would it be possible to use a shared cq for all the eps on the cm?

Reply:
@evgeny-leksikov creating a cq on the cm would mean to 'attach' it to some device since would need to provide a context for the cq. keeping it on the ep would allow us to use the cm_id of the ep which already holds an ibv_context.
also, now i remembered that we already discussed it with @yosefe and said that if the cq will be small enough, it shouldn't consume a lot of resources. (use only one entry)
cq = ibv_create_cq(id->verbs, 1, NULL, NULL, 0);
WDYT?